src/Microsoft.TestPlatform.Build/Tasks/VSTestForwardingApp.cs

```diff
@@ -33,23 +33,15 @@ public int Execute()
                                       FileName = hostExe,
                                       Arguments = string.Join(" ", this.allArgs),
                                       UseShellExecute = false,
-                                      CreateNoWindow = true,
```

Comment:
@codito `dotnet test` working fine for above three scenarios. No color is displaying for `dotnet test` on VSTS task (same behavior as `dotnet build`).

@shanselman Yes, `dotnet watch test` displaying colored output.